### ISSUE TITLE
Can i got private key which derived by Nist256p1 ?

### ISSUE THREAD

**Issue by weixuefeng** [MetaMask/key-tree#57]
```
public static DeterministicKey createMasterPrivateKeyNist(byte[] seed) throws HDDerivationException {
        return createMasterPrivateKeyWithKeys(seed, "Nist256p1 seed".getBytes());
    }
```

**Comment by rekmarks** [comment]
Hey @weixuefeng, thank you for creating this issue! Could you tell us a bit more about your use case? Specifically, why do you need to use the NIST curve? Is it because your protocol requires it?

**Comment by weixuefeng** [comment]
yes, i want to add "NEW" coin by snap, [newton](https://github.com/newtonproject/newchain-sdk-example), NEO, ONT used NIST algorithm. could you help me @rekmarks thanks for you reply!

**Comment by rekmarks** [comment]
@weixuefeng we are currently blocked on adding this capability due to the lack of a good JavaScript implementation of this curve. We could _maybe_ make do with WebAssembly, but we do not have the capacity to research the matter at this time.

I will keep this issue open since it is a legitimate use case that we want to support.

**Comment by weixuefeng** [comment]
ok

**Comment by paulmillr** [comment]
I would argue against this. This would make key-tree bigger...the use case is not defined/popular enough
